ppr-ui/tests/unit/TransportPermitDetails.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { TransportPermitDetails } from '@/components/mhrTransportPermit'
+import { createComponent } from './utils'
+import { nextTick } from 'vue'
+import { beforeEach } from 'vitest'
+import { useStore } from '@/store/store'
+import { MhApiStatusTypes } from '@/enums'
+import flushPromises from 'flush-promises'
+
+const store = useStore()
+
+describe('Transport Permit Details', () => {
+  let wrapper
+
+  beforeEach(async () => {
+    store.setMhrInformation({
+      permitDateTime: '2024-02-05T08:40:53-08:00',
```

Comment:
Wouldn't it start to fail after the date has passed?

Reply:
This is just hard corded mock data, will always be the same.
Intention was to test for the formatted output in the template.